components/docs-page.js

```diff
@@ -122,7 +122,14 @@ function DocSection(props) {
           dangerouslySetInnerHTML={{__html: mdToHTMLUnwrapped(title)}}
         />
       </Anchor>
-      <h4 dangerouslySetInnerHTML={{__html: mdToHTMLUnwrapped(subtitle)}} />
+      <glamorous.Div
```

Comment:
A `div` doesn't seem semantically accurate either. 😕 Maybe don't use `mdToHTMLUnwrapped` so we at least get a `p` tag? I could be missing something though...